tests/queries/0_stateless/03286_parallel_replicas_cross_join_bug.sql

```diff
@@ -0,0 +1,9 @@
+drop table if exists tab;
+create table tab (x UInt64) engine = MergeTree order by tuple();
+insert into tab select number from numbers(1e7);
+
+set enable_parallel_replicas = 1, max_parallel_replicas = 3, cluster_for_parallel_replicas = 'parallel_replicas', parallel_replicas_for_non_replicated_merge_tree = true, parallel_replicas_local_plan = false;
```

Comment:
Why disable local plan (the setting is randomized, btw)?

Reply:
That was just a subset of settings that did reproduce the issue. Let me remove the one with local plan.